packages/npm-packages/ruby-wasm-wasi/tools/run-test-unit.mjs

```diff
@@ -169,6 +169,18 @@ const test = async (instantiate) => {
 
   await vm.evalAsync(`
     require 'test/unit'
+
+    # Define the methods to be used for unit testing assertions.
+    # Use refinements to limit the scope of influence.
+    require 'pp'
+    module JsObjectTestable
+        refine JS::Object do
+          [:object_id, :pretty_inspect].each do |method|
```

Comment:
If those definitions are just for testing, it might be better to fix testing libraries in the long term. I just opened a test-unit fix for BasicObject support. https://github.com/test-unit/test-unit/pull/262

But it's fine for now. Could you add FIXME comments mentioning the fix PR and potential future fix?